packages/elastic-datemath/package.json

```diff
@@ -14,12 +14,12 @@
     "@babel/cli": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
     "babel-plugin-add-module-exports": "^1.0.2",
-    "moment": "^2.13.0"
+    "moment": "^2.24.0"
   },
   "dependencies": {
     "tslib": "^1.9.3"
   },
   "peerDependencies": {
-    "moment": "^2.13.0"
+    "moment": "^2.24.0"
```

Comment:
Can we remove the peerDependency here if it's already a dependency?

Reply:
It's a dev dep, and part of this package involves mutating the package so I think it's best to make sure that it's a peer and not a direct dependency.

Reply:
You're right, missed that.